tests/extension/khr_max_num_work_groups/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(SYCL_CTS_ENABLE_MAX_NUM_WORK_GROUPS)
```

Comment:
I am confused. Should it be `SYCL_CTS_ENABLE_KHR_MAX_NUM_WORK_GROUPS` instead?

Reply:
Looks like a bug vs line 130